app/Domain/Integrations/Models/IntegrationModel.php

```diff
@@ -201,6 +201,11 @@ public function toDomain(): Integration
                 ->get()
                 ->map(fn (IntegrationUrlModel $integrationUrlModel) => $integrationUrlModel->toDomain())
                 ->toArray()
+        )->withAuth0Clients(
```

Comment:
@simon-debruijn if I remember correctly the Auth0 clients are retrieved in a separate call. But with this changed they are always embedded.